great-idea-website/index.html

```diff
@@ -14,50 +14,101 @@
   <![endif]-->
 </head>
 
+<!-- I did it!!! -->
   <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
+  
+ <div ="navTop">
+  <header>
+  
+<nav>
+  <ul>
+  <li><a href="#">Services</a> </li>
+  <li><a href="#">Product</a></li>
+  <li><a href="#">Vision</a></li>
+  <li><a href="#">Features</a></li>
+  <li><a href="#">About</a></li>
+  <li><a href="#">Contact</a></li>
+  </ul>
+</nav>
 
   <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
+  
+  </header>
+  
+ </div>
+ 
+ <div class="container">
+  
+  
+  <div class="sectionOne">
+  
+  <div class="iod">
+	<h1>
+ Innovation <br>
+  On <br>
+  Demand <br>
+  </h1>
+  
+  <div class="button1">
   Get Started
+  </div>
+  </div>
+  
+  
+  
 
-  <img src="img/header-img.png" alt="Image of a code snippet.">
+  <img class="circle" src="img/header-img.png" alt="Image of a code snippet.">
+  </div>
+  
+  
+  
 
-  Features
+  <div class="features">
+ <strong> Features </strong> <br> <br>
```

Comment:
Instead of using line breaks for spacing, try to achieve the same appearance with margins + padding. 😃